src/framework/piral-core/src/modules/api.ts

```diff
@@ -111,8 +111,7 @@ export function createCoreApi(context: GlobalStateContext): PiletApiExtender<Pil
         context.unregisterExtension(name as string, arg);
       },
       renderHtmlExtension(element, props) {
-        const [dispose] = render(context, element, props);
-        return dispose;
+        return render(context, element, props);
```

Comment:
This is a breaking change - we will not do that. Instead, use the DOM event to propagate the update (extension-props-changed).

Reply:
Ok👌